CRM/Contact/Form/Relationship.php

```diff
@@ -299,9 +299,6 @@ public function buildQuickForm() {
     // Select list
     $relationshipList = CRM_Contact_BAO_Relationship::getContactRelationshipType($this->_contactId, $this->_rtype, $this->_relationshipId);
 
-    // Metadata needed on clientside
-    $this->assign('relationshipData', self::getRelationshipTypeMetadata($relationshipList));
```

Comment:
It would be nice to clean it up to avoid having code that does the same thing in JS and PHP, but it's still used in [one other place](https://github.com/civicrm/civicrm-core/blob/master/CRM/Case/Form/CaseView.php#L197). My JS skills are very limited, but if you think it's best that we remove this function and provide some common service to fetch this data then I can give that a shot.

Reply:
Sorry, didn't realize it was used elsewhere. I don't think that's a priority.